### ISSUE TITLE
 enhanced usage-script

### ISSUE THREAD

**Issue by DavidBruchmann** [jitesoft/php-xml2array#6]
```
<?php
/**
 * Skript to test or use https://github.com/jitesoft/php-xml2array/
 *
 */

$xmlFile = '';  # <<< Insert your XML-file here

$parserPath = 'jitesoft/php-xml2array/src/' ; # <<< Adjust the path here

// variable for the PHP-Array
$arrayVariable = '$myArray';  # <<< Adjust variable-name

// Filename for Json (includ path if required)
$jsonFile = 'test.jitesoft.xml2array.json';  # <<< Adjust file-name

// Filename for PHP-file with resul-array (includ path if required)
$phpFile = 'test.jitesoft.xml2array.php';  # <<< Adjust file-name

### Everything adjusted above?
### Run the script!

require_once($parserPath.'ParserInterface.php');
require_once($parserPath.'Parser.php');
require_once($parserPath.'Node.php');


function this_var_export($var, $return=FALSE, $tabsforSpaces=TRUE){
	$out = var_export($var,1);
	if($tabsforSpaces) {
		$out = str_replace('  ',"\t",$out);
	}
	// Reformating a bit:
	$out = preg_replace('/=>\s*array \(/ms','=> array(',$out);
	$out = preg_replace('/\(\s*\)/ms','()',$out);
	if(!$return)
		echo '<pre>'.$out.'</pre>';
	else
		return $out;
}

// ( It didn't work with only one instance: )
# $parser = new Jitesoft\XML\Parser();
$parser2 = new Jitesoft\XML\Parser();
$parser3 = new Jitesoft\XML\Parser();
$xmlContent = file_get_contents($xmlFile);

# // Node tree!     same like Json??
# $out    = $parser->parse($xmlContent);
# file_put_contents('test.jitesoft.nodeTree',$out);

// Creating and saving Json string in file
$out2    = $parser2->parse($xmlContent, $parser2::OUT_TYPE_JSON);
file_put_contents($jsonFile, $out2);

// Creating and saving PHP-array in file
$out3    = $parser3->parse($xmlContent, $parser3::OUT_TYPE_ARRAY);
$out3    = $arrayVariable.' = ' . this_var_export($out3,1) . ';' . "\n";
file_put_contents($phpFile, '<?php' . "\n\n" . $out3 . "\n" . '?>');

// Output PHP-Array also on screen:
echo '<pre>' . "\n" . $out3 . '</pre>' . "\n";

```

Next step would be a form for the configration ...

**Comment by DavidBruchmann** [comment]
Above script is for a cloned version, but it's stated that it should be installed by composer.
Then the file has to look different.

**Comment by DavidBruchmann** [comment]
perhaps providing several example-scripts or opening the wiki ;-)

**Comment by Johannestegner** [comment]
Good idea, I will take care of this asap! :)

**Comment by Johannestegner** [comment]
Fixed with 25518f707a2d62c93399d7836374637899ce353f
